tysnj/fitlit
# Feature/calendar

## Is this a fix or a feature?
Feature

## What is the change?
Datepicker functionality with calendar

## What does it fix?
Default date with calendar functionality
Updating visualizations when dates are chosen.

## Where should the reviewer start?
fetch feature/calendar branch

## How should this be tested?
run `open src/index.html` and visually confirm.
change some dates and check functionality.
